src/test/regress/sql/multi_colocation_utils.sql

```diff
@@ -457,14 +459,6 @@ SELECT create_distributed_table('range_table', 'a', 'range');
 
 SELECT create_reference_table('ref');
 
-
--- check d1, d2, d3 and d4 has the same colocation id => they are colocated.
-SELECT get_table_colocation_id('d1');
```

Comment:
It seemed that these were not really useful as we are already checking the colocations with `tables_colocated` function.

Also my guess on why the colocation id was different for pg 11 and (pg12,13) is that there were the same entries in pg_dist_colocation for colocation id 4 and 5. And while reading from the table pg11 might be returning 5 whereas pg12 is returning 4. 

This brings one question to me, is it possible that creating two tables with `create_distributed_table('table1', 'a');` and `create_distributed_table('table2','a');` would put them into different colocations.